docs/en/guides/apply_catboost_model.md

```diff
@@ -0,0 +1,213 @@
+
+Before applying a CatBoost model:
+
+**1.** Pull the [Docker image](https://hub.docker.com/r/yandex/tutorial-catboost-clickhouse) from the registry:
+
+```bash
+$ docker pull yandex/tutorial-catboost-clickhouse
+```
+
+This Docker image contains everything you need to run CatBoost and ClickHouse: code, runtime, libraries, environment variables, and configuration files.
+
+**2.** Make sure the Docker image has been successfully pulled:
+
+```bash
+$ docker image ls
+REPOSITORY                            TAG                 IMAGE ID            CREATED             SIZE
+yandex/tutorial-catboost-clickhouse   latest              622e4d17945b        22 hours ago        1.37GB
+```
+
+**3.** Start a Docker container based on this image:
+
+```bash
+$ docker run -it -p 8888:8888 yandex/tutorial-catboost-clickhouse
+```
+
+## 1. Create a Table {#create-table}
+
+To create a ClickHouse table for the train sample:
+
+**1.** Start a ClickHouse client:
+
+```bash
+$ clickhouse client
+```
+
+!!! note "Note"
+    The ClickHouse server is already running inside the Docker container.
+
+**2.** Create the table using the command:
+
+```sql
+:) CREATE TABLE amazon_train
+(
+    date Date MATERIALIZED today(), 
+    ACTION UInt8, 
+    RESOURCE UInt32, 
+    MGR_ID UInt32, 
+    ROLE_ROLLUP_1 UInt32, 
+    ROLE_ROLLUP_2 UInt32, 
+    ROLE_DEPTNAME UInt32, 
+    ROLE_TITLE UInt32, 
+    ROLE_FAMILY_DESC UInt32, 
+    ROLE_FAMILY UInt32, 
+    ROLE_CODE UInt32
+)
+ENGINE = MergeTree()
+```
+
+## 2. Insert the Data to the Table {#insert-data-to-table}
+
+To insert the data:
+
+**1.** Exit from ClickHouse console client:
+
+```sql
+:) exit
+```
+
+**2.** Upload the data:
+
+```bash
+$ clickhouse client --host 127.0.0.1 --query 'INSERT INTO amazon_train FORMAT CSVWithNames' < ~/amazon/train.csv
+```
+
+**3.** Make sure the data has been uploaded:
+
+```sql
+$ clickhouse client
+:) SELECT count() FROM amazon_train
+
+SELECT count()
+FROM amazon_train
+
++-count()-+
+|   65538 |
++---------+
+```
+
+## 3. Configure the Model to Work with the Trained Model {#configure-model}
+
+**Optional step**: the Docker container contains model configuration file and link to it in ClickHouse configuration. See `models/amazon_model.xml` and `../../etc/clickhouse-server/config.d/models_config.xml`.
+
+**1.** Create a model config file in the `models` folder (for example, `models/config_model.xml`):
+
+```xml
+<models>
+    <model>
+        <!-- Model type. Now catboost only. -->
+        <type>catboost</type>
+        <!-- Model name. -->
+        <name>amazon</name>
+        <!-- Path to trained model. -->
+        <path>/home/catboost/tutorial/catboost_model.bin</path>
+        <!-- Update interval. -->
+        <lifetime>0</lifetime>
+    </model>
+</models>
+```
+
+**2.** Add a link to the created file to the ClickHouse configuration `../../etc/clickhouse-server/config.d/models_config.xml`.
```

Comment:
* it looks like it's not a link, but a file that imports another file
* why do we need `../..`? it looks like an absolute path and we could safely drop this prefix